site/src/docs/about/releases.mdx

```diff
@@ -0,0 +1,39 @@
+---
+slug: '/about/releases'
+title: 'Releases'
+---
+
+import ExternalLink from '../../components/ExternalLink';
+import LeadParagraph from '../../components/LeadParagraph';
+import InternalLink from '../../components/InternalLink';
+
+
+# Releases
+
+## Coming up
+
+Next quarter we are working on these larger features and fixes. Smaller feature updates will be coming up regularly.
+
+### Cookie consent
+- design updates to simplify the user experience
+- core styles for the banner and page
+- bug fixes
+- investigations on web components and js hooks to provide the cookie management to different platforms
+
+### Dropdown refactoring
+- design updates to make it clearer
+- removing outdated downshift-library
+- improving accessibility
+- bug fixes
+- refactor code
+
+## Latest release
+
+<ExternalLink href="https://github.com/City-of-Helsinki/helsinki-design-system/releases/latest">Latest release </ExternalLink>
+    - <ExternalLink href="https://github.com/City-of-Helsinki/helsinki-design-system/releases/latest/download/hds-design-kit.zip">Design kit</ExternalLink>
+    - <ExternalLink href="https://github.com/City-of-Helsinki/helsinki-design-system/releases/latest/download/hds-favicon-kit.zip">Favicon kit </ExternalLink>
+    - <ExternalLink href="https://github.com/City-of-Helsinki/helsinki-design-system/releases/latest/download/hds-icon-kit.zip">Icon kit </ExternalLink>
+
+## All previous releases  
+
+See previous releases and their release notes in <ExternalLink href="https://github.com/City-of-Helsinki/helsinki-design-system/releases" >GitHub </ExternalLink>
```

Comment:
I would say this needs `openInNewTab` property 🤔

Reply:
Added